### PR TITLE
Responsive event package show

### PR DESCRIPTION
**Pull request by elbyrd09**
Booking journey responsiveness

### CODE REVIEW

File: app/assets/stylesheets/components/_photographer_carousel.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.carousel.pointer-event {
```
Comment (Matth93) [inline app/assets/stylesheets/components/_photographer_carousel.scss]: What doeds the pointer event class do? And when/where is it used?

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -12,3 +12,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+# Rails.application.config.assets.precompile += %w( devise_user.scss )
```
Comment (Matth93) [inline config/initializers/assets.rb]: Is this how you precompile assets also in Heroku? lol, that's been giving me a hard time :P

File: run.save [hunk 3]
```diff
@@ -0,0 +1,3 @@
+show-method run
```
Comment (Matth93) [inline run.save]: Just curious: what's this?

File: app/views/packages/show.html.erb [hunk 4]
```diff
@@ -104,59 +105,51 @@
...
       slot.classList.add("available-slot");
     }
-
     slot.addEventListener('click', updateForm);
     if (slot.classList.contains('available-slot')) {
       slot.addEventListener('click', changeColor);
     };
-
   })
 </script>
```
Comment (Matth93) [inline app/views/packages/show.html.erb]: You got it working? Looks good! But do try to have everything lined up nicely, like the divs of 'carousel-item' and 'photographerModal'. should start on the same column. I've found that (in Sublime) Edit > Line > Reindent can definitely help. Only it doesnt indent Ruby iterations, which still have to be done. This also made me notice you are probably missing a final closing div for the main container?

File: app/assets/stylesheets/components/_photographer_carousel.scss (hunk 1 above)
Comment (elbyrd09) [reply app/assets/stylesheets/components/_photographer_carousel.scss]: it's used on the package show page.  I made a new component file for it but can likely take the styling for pointer-event and add it to the _carousel.scss file  @Matth93

File: config/initializers/assets.rb (hunk 2 above)
Comment (elbyrd09) [reply config/initializers/assets.rb]: yeah, took a lot of googling to figure it out finally

File: run.save (hunk 3 above)
Comment (elbyrd09) [reply run.save]: no clue , must have been generated automatically somehow

File: app/views/packages/show.html.erb (hunk 4 above)
Comment (elbyrd09) [reply app/views/packages/show.html.erb]: I'll count the divs again

**Pull request merged**